components/transfer/Transfer.vue

```diff
@@ -258,7 +256,6 @@ export type TargetAddress = {
 const transactionValue = ref('')
 const price = ref(0)
 const usdValue = ref(0)
-const targets = ref(emptyObject<TargetMap>())
```

Comment:
@Jarsen136 any reason why this got removed , I think we do need it 
![CleanShot 2023-08-18 at 11 23 53@2x](https://github.com/kodadot/nft-gallery/assets/44554284/3648dc46-b60e-41fe-a191-b88b45955132)

Reply:
These codes with old logic stay at `components/transfer/TransferOld.vue`. We are going to redesign them. Thanks for pointing out.